transforms/handle_the_DC.xsl

```diff
@@ -17,7 +17,7 @@
     <xsl:template match="dc:dc">
         <xsl:copy>
             <xsl:apply-templates/>
-            <identifier type="hdl">
+            <dc:identifier xmlns:dc="http://purl.org/dc/elements/1.1/" type="hdl">
                 <xsl:value-of select="$handle_value"/>
             </identifier>
```

Comment:
Closing should probably also get the prefix. Actually... Adding the prefix here shouldn't be necessary at all, as it's the [default namespace for this stylesheet](https://github.com/discoverygarden/islandora_handle/pull/13/files#diff-61cb4fe92331890bd19753ca905d2b2fR4). Shrug?